tools/cli/workflowCommands.go

```diff
@@ -311,6 +316,35 @@ func processSearchAttr(c *cli.Context) map[string][]byte {
 	return fields
 }
 
+func processHeader(c *cli.Context) map[string][]byte {
```

Comment:
This looks similar to processMemo, can we try refactor to reduce duplicates?

Reply:
Sure, refactored them.